docs/serving/using-a-custom-domain-per-service.md

```diff
@@ -0,0 +1,66 @@
+
+---
+title: "Setting up a custom domain per Service"
+weight: 55
+type: "docs"
+---
+
+By default, Knative uses the `{route}.{namespace}.{default-domain}` fully qualified domain name for the Service, where `default-domain` is `example.com`. You are able to change the `default-domain` following the [Setting up a custom domain](./using-a-custom-domain.md) guide.
+
+This guide documents the process to use a custom FQDN for a Service, like `my-service.example.com`, created by [@bsideup](https://bsideup.github.io/posts/knative_custom_domains/).
+
+> There is currently no formal process to set up a custom domain per Service. The topic is being discussed [here](https://github.com/knative/serving/issues/2985).
+
+## Edit using kubectl
+
+1. Edit the `domainTemplate` entry on the `config-network` configuration:
+
+   ```shell
+   kubectl edit cm config-network --namespace knative-serving
+   ```
+
+   Replace the `domainTemplate` with the following (the spaces must be respected):
```

Comment:
Consider adding a link to https://github.com/knative/serving/blob/3e920bd986ecf6d22d506a273a8ed15c4f266eec/config/core/configmaps/network.yaml#L69 for a wider explanation of available options

Reply:
Will do. Will use the master branch, tho :)